content/src/denylist/Denylist.ts

```diff
@@ -101,8 +113,9 @@ export class Denylist {
           this.network
         )
       })
+      return { status: OperationStatus.OK }
```

Comment:
I think we can avoid the try/catch here. Something like:
```
return new Promise((resolve) => {
validateSignature(
          metadata,
          messageToSign,
          resolve({ status: OperationStatus.OK }),
          errorMessage => resolve({ status: OperationStatus.ERROR, message: `Failed to authenticate the blocker. Error was: ${error}` }),
          (signer) => !!signer && (nodeOwner === signer || this.authenticator.isAddressOwnedByDecentraland(signer)),
          this.network
        )
})        
```

Reply:
niceeee